engine/src/main/java/org/terasology/rendering/dag/WireframeTask.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag;
+
+import org.lwjgl.opengl.GL11;
+import static org.lwjgl.opengl.GL11.GL_FILL;
+import static org.lwjgl.opengl.GL11.GL_FRONT_AND_BACK;
+import static org.lwjgl.opengl.GL11.GL_LINE;
+
+/**
+ *
+ */
+public class WireframeTask implements PipelineTask { // TODO: separate this into SetWireframeOnTask and SetWireframeOffTask?
```

Comment:
Yes, you mentioned that earlier. For not dealing with "bigger can of worms" at least I can do is still not put the check inside Node's `process()`, but inside `WorldRendererImpl` to track whether configuration is changed or not. This would eliminate several ifs for now. Later I want to deal with those in the future!